e2e/install_test.go

```diff
@@ -2255,3 +2257,97 @@ func TestInstallWithPrivateCAs(t *testing.T) {
 
 	t.Logf("%s: test complete", time.Now().Format(time.RFC3339))
 }
+
+func TestInstallWithConfigValues(t *testing.T) {
```

Comment:
Is there a way to test this without e2e?

Reply:
not that I'm aware of, because the thing being tested is "does kots work in this scenario" (and considering that it panics without code changes that is a very relevant thing to test)

Reply:
hmm, i think you can use:
- the new dryrun integration testing framework to verify that the config values get passed down to the kots cli correctly
- write a unit or integration test in kots for determining the next app status
?

it is a bit more effort

Reply:
It's a bit more effort and doesn't actually cover the code flow in question
I can add an integration test to make sure the values are getting passed properly, but adding a test in kots that covers this flow is a lot harder - the function in question is not unit tested there and we do not have a integration test that uses embedded cluster

Reply:
I've added this integration test, but it doesn't cover much (just that we're actually passing the flag to kots)